tests/requirements-libraries.txt

```diff
@@ -51,28 +52,23 @@ ipython==7.16.1; python_version == '3.6'  # pyup: ignore
 ipython==7.9.0; python_version == '3.5'  # pyup: ignore
 
 # pandas 1.0.0 dropped Python 3.5 support.
-pandas==1.1.3; python_version > '3.5'
+# Not yet Python 3.9 compatible because of Numpy
+pandas==1.1.3; python_version > '3.5' and python_version < '3.9'
 pandas==0.25.3; python_version == '3.5'  # pyup: ignore
 
 # Numpy 1.19 dropped Python 3.5 support
-numpy==1.19.2; python_version > '3.5'
+numpy==1.19.2; python_version > '3.5' and python_version < '3.9'
 numpy<=1.19.0; python_version == '3.5'  # pyup: ignore
 
 # And so did SciPy
-scipy==1.5.3; python_version > '3.5'
+# Not yet Python 3.9 compatible because of Numpy
+scipy==1.5.3; python_version > '3.5' and python_version < '3.9'
 scipy<=1.5.0; python_version == '3.5'  # pyup: ignore
 
 # Pillow 8.0 dropped support for python 3.5
 Pillow==8.0.0; python_version > '3.5'
 Pillow<8.0.0; python_version == '3.5' # pyup: ignore
 
-# Special cases
-# -------------
-
-# Per https://www.riverbankcomputing.com/pipermail/pyqt/2020-February/042506.html, Windows PyQt5 5.14.0? and 5.14.1? aren't correctly packaged.
-pyqt5==5.13.2; sys_platform == 'win32'
-pyqt5==5.14.1; sys_platform != 'win32'
-# On Appveyor, newer versions fail with the error message
-# ``[1916:2732:0204/144922.809:ERROR:mf_helpers.cc(14)] Error in dxva_video_decode_accelerator_win.cc on line 513``
-pyqtwebengine==5.12.1; sys_platform == 'win32'  # pyup: ignore
-pyqtwebengine==5.14.0; sys_platform != 'win32'  # pyup: ignore
```

Comment:
Wha have all these spcaial cases been removed?

Reply:
Tests are passing on Windows, so yes :)
I am still working to make this OK on GNU/Linux though (draft PR).

Reply:
The Qt stuff _should_ be fine if you upgrade the versions, but check.